src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -121,14 +122,28 @@ public XmlSchema GetIntersectionSchema(string ns1, string ns2, string attrNs)
             return XmlSchema.Read(new StringReader(xsd), null);
         }
 
+            public XmlSchema GetSimpleSchema(string ns, string attrNs)
+            {
+                var xsd = @"<xs:schema xmlns:xs='http://www.w3.org/2001/XMLSchema' targetNamespace='" + attrNs + @"' xmlns='" + attrNs + @"'>
+                        <xs:complexType name = 't'>
+                            <xs:sequence >
+                            <xs:element name = 'e1' type = 'xs:string' />
+                            </xs:sequence >
+                            <xs:anyAttribute namespace='" + ns + @"'/>
+                        </xs:complexType>
+                        </xs:schema>";
+
+                return XmlSchema.Read(new StringReader(xsd), null);
+            }
+
         //Intersection namespaces
         [Theory]
         //[Variation(Desc = "complextype Any ns - ##any, attrgroup Any ns2, allow ns2 attribute")]
-        [InlineData("##any", "ns2", "ns2", 0)]
+        [InlineData("##any", "ns2", "ns2", 0, "##targetNamespace")]
```

Comment:
since the behavior is now different on full framework and corefx you might solve it in two ways:
- split test for generic tests and corefx specific and put XYZ attribute on corefx specific
- use PlatformDetection.IsFullFramework for those tests (i.e. additional bool arg) and `throw new TestSkippedException()`

some usage examples:
https://github.com/dotnet/corefx/blob/8c7771a74510479afc7417041cb5208354164ff6/src/Microsoft.Win32.SystemEvents/tests/SystemEvents.CreateTimer.cs#L50
https://github.com/dotnet/corefx/blob/5ae3c5655cd84bf3c7f9568477153642fdd7c95e/src/System.Reflection/tests/AssemblyTests.cs#L148
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cookies.cs#L217

Reply:
Thanks for the pointers, made it easy to fix.
I'll try to look at the failing Linux arm64 tests tomorrow.